src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala

```diff
@@ -587,11 +596,14 @@ object SbtBundle extends AutoPlugin {
   private def formatServiceName(serviceName: String): String =
     s"""        service-name  = "$serviceName""""
 
-  private def formatEndpoints(endpoints: Map[String, Endpoint]): String = {
+  private def formatEndpoints(bundleConfVersion: BundleConfVersions.Value, endpoints: Map[String, Endpoint]): String = {
     val formatted =
       for {
         (label, Endpoint(bindProtocol, bindPort, services, serviceName, acls)) <- endpoints
       } yield {
+        if (acls.exists(_.nonEmpty) && bundleConfVersion != BundleConfVersions.V_1_2_0)
+          throw new IllegalArgumentException("Request ACL may only be specified for bundle.conf version 1.2.0")
```

Comment:
What about a validation for `1.1`? We should throw an exception if bundleConfVersion 1.1 is used and the acls are specified and if the `services` are not specified.